pkg/adapter/runtime.go

```diff
@@ -369,3 +369,39 @@ func (r *LocalRuntime) Diff(c *cliconfig.DiffValues, to string) ([]archive.Chang
 func (r *LocalRuntime) GenerateKube(c *cliconfig.GenerateKubeValues) (*v1.Pod, *v1.Service, error) {
 	return shared.GenerateKube(c.InputArgs[0], c.Service, r.Runtime)
 }
+
+// GetPodsByStatus returns a slice of pods filtered by a libpod status
+func (r *LocalRuntime) GetPodsByStatus(statuses []string) ([]*Pod, error) {
+	var adapterPods []*Pod
+
+	filterFunc := func(p *libpod.Pod) bool {
+		state, _ := shared.GetPodStatus(p)
+		for _, status := range statuses {
```

Comment:
Why cannot Go do generics?  Python's 'in' would be perfect here...

Reply:
because go is not magic like python is...

Reply:
We'll get... something similar... in Go 2. Probably. It won't be as good as an actual modern language like Rust, but it'll be better than living in the dark ages like we are now.